modules/lvms-creating-volume-snapshots.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-volume-snapshots_{context}"]
+= Creating volume snapshots
+
+You can create volume snapshots based on the available capacity of the thin pool and the over-provisioning limits.
+To create a volume snapshot, you must create a `VolumeSnapshotClass` object.
```

Comment:
`VolumeSnapshotClass` object always gets created by lvms for thinPool, I think we can skip this line saying 'you must create a VolumeSnapshotClass object'.

Reply:
Agreed!
Here, I actually intended to write `VolumeSnapshot` object. Updated it to `VolumeSnapshot`.